extension/extension/background.js

```diff
@@ -237,8 +238,12 @@ function log_event(event_type, event) {
     }
 }
 
+function logFromServiceWorker(event) {
```

Comment:
For all functions you should add a comment indicating the purpose of the function and the expected value.

Reply:
I've added this